indexing-hadoop/src/main/java/org/apache/druid/indexer/HadoopDruidIndexerConfig.java

```diff
@@ -138,6 +138,20 @@
     ROWS_THROWN_AWAY_COUNTER
   }
 
+  public static Map<String, String> getAllowedProperties(List<String> listOfAllowedPrefix)
```

Comment:
`listOfAllowedPrefix` seems to always be the `allowedHadoopPrefix` of this class. Can it be a non-static method?

Reply:
JobHelper.injectSystemProperties and JobHelper.injectDruidProperties methods are static. If we change those to non-static then yes, getAllowedProperties can be non-static since listOfAllowedPrefix are always allowedHadoopPrefix. But maybe leaving it like this is more flexible as the caller might pass something other than allowedHadoopPrefix for listOfAllowedPrefix

Reply:
Leaving as is unless you have strong opinion on changing all those to non-static and having JobHelper instances. @jihoonson

Reply:
I guess you can make this method non-static while keeping those methods if they accept `HadoopDruidIndexerConfig` instead of `listOfAllowedPrefix`?

Reply:
I like that. Done